packages/lesswrong/server/serverStartup.ts

```diff
@@ -144,6 +145,7 @@ export const serverStartup = async () => {
 
     cluster.on('exit', (worker, _code, _signal) => {
       console.log(`Worker ${worker.process.pid} died`);
+      setTimeout(() => cluster.fork(), processRestartDelay);;
```

Comment:
Two semicolons.  Also, are we very sure that we're only going to get one `exit` signal from any given worker?  Otherwise lgtm

Reply:
This pattern appeared (sans delay timer) in a code sample in the documentation for `node:cluster`, so I think it does make that guarantee.